fbw-a380x/src/systems/instruments/src/EWD/elements/ThrustRatingMode.tsx

```diff
@@ -20,9 +20,25 @@ export class N1Limit extends DisplayComponent<{
 }> {
   private readonly N1LimitType = ConsumerSubject.create(null, 0);
   private readonly N1ThrustLimit = ConsumerSubject.create(null, 0);
+  private readonly N1ThrustLimitIdle = ConsumerSubject.create(null, 0);
+  private readonly N1ThrustLimitToga = ConsumerSubject.create(null, 0);
   private readonly flexTemp = ConsumerSubject.create(null, 0);
   private readonly sat = Arinc429ConsumerSubject.create(undefined);
   private readonly thrustLimitTypeArray = ['', 'CLB', 'MCT', 'FLX', 'TOGA', 'MREV'];
+  private readonly thrIdleOffset = 0.042;
```

Comment:
What does this value represent? Please add a JSDoc with it's units/where it comes from.

Reply:
comment will be added. It's a value to offset idle thrust taken from ThrustGauge.tsx. Where is a good place to put such constants?